app/assets/scss/_search_overview.scss

```diff
@@ -18,7 +18,7 @@ ol.instruction-list-item-top li {
 
 ol.steps {
   @include bold-27;
-  padding-left: 25px;
+  padding-left: 30px;
```

Comment:
I'm not sure I understand why we would want to increase this. The briefs-FE is 25px (albeit approximated via `em`s) so we stay consistent with that. It also doesn't help with either of the issues for this PR: getting the step text aligned with the paragraph, or getting the step numbers to have equal spacing.

Reply:
due to an increased size of font used in the direct award task list, 25px is sometimes not enough to make the rows consistent.

Reply:
Should we instead bring the fontsize back down to whatever the briefs one is to be more consistent then?

Reply:
+1 on consistency here